javalib/src/main/scala/java/lang/dec2flt/FloatingPoint.scala

```diff
@@ -0,0 +1,152 @@
+/*
+ * Scala.js (https://www.scala-js.org/)
+ *
+ * Copyright EPFL.
+ *
+ * Licensed under Apache License 2.0
+ * (https://www.apache.org/licenses/LICENSE-2.0).
+ *
+ * See the NOTICE file distributed with this work for
+ * additional information regarding copyright ownership.
+ */
+
+package java.lang.dec2flt
+
+import java.math.BigInteger
+import java.lang.{Long => jlLong}
+
+/** DIY floating point number with 64-bit significand bits */
```

Comment:
```suggestion
/** DIY floating point number with 64-bit significant bits */
```
(with a 't' if used as an adjective; with a 'd' if used as a noun)